source/resourceManager.cpp

```diff
@@ -115,10 +116,215 @@ int resourceManager(bool populated, int departmentCounter, string fileStoragePat
 		
 		departmentList.push_back(departmentObject);
 	}
-
+	outToIn();
```

Comment:
Why is this being called in this function? I believe it should only be called by the validate button on the GUI.